### ISSUE TITLE
Workspace labels not working?

### ISSUE THREAD

**Issue by Laxen** [mzur/gnome-shell-wsmatrix#100]
Hello! 

I would like to have labels instead of thumbnails on the workspace switcher but I can't get it to work. I looked at the [wiki](https://github.com/mzur/gnome-shell-wsmatrix/wiki/Assigning-custom-labels-to-workspaces) and have put in my workspace names and made sure I have a string for each workspace, but it still only shows thumbnails. I found the code that I believe enables/disables workspace names [here](https://github.com/mzur/gnome-shell-wsmatrix/blob/b395025af90a2b04103320111b962407be3fe77f/wsmatrix%40martin.zurowietz.de/WmOverride.js#L235). Looking at this it seems to want a "show-workspace-names" boolean in dconf, so I added that (to /org/gnome/shell/extensions/wsmatrix/). But still no cigar.

```
$ dconf dump /org/gnome/shell/extensions/wsmatrix/
[/]
num-rows=3
show-workspace-names=true
```
```
$ dconf dump /org/gnome/desktop/wm/preferences/
[/]
num-workspaces=9
workspace-names=['I', 'like', 'to', 'have', 'workspace', 'labels', 'thank', 'you', 'friend']
```

I've logged out and in after changing the settings. Is there something obvious I'm missing here?

Thanks in advance!

**Comment by ebeem** [comment]
you can use gnome tweaks and configure the extension from its own preferences window
![Screenshot from 2020-03-16 07-00-35](https://user-images.githubusercontent.com/25146039/76722264-e4651e00-6753-11ea-9853-9245069fc232.png)

make sure you `disable` `show workspace thumbnails` and `enable` `show workspace names`

**Comment by Laxen** [comment]
Hmm, that's weird. I assumed that window didn't work because mine looks like this:
![Selection_058](https://user-images.githubusercontent.com/1144747/76730539-c6032080-675b-11ea-8acc-123be46e7ba8.png)

For the record I'm running gnome-shell version 3.28.4 and I've installed the extension from here: https://extensions.gnome.org/extension/1485/workspace-matrix/. I chose shell version 3.28 and extension version 11. 

I just realized that gnome-shell 3.34 has extension version 17 available. So maybe I'm just running a really old version that doesn't support this. I'll update my stuff and try version 17 tonight!

Thanks for the super fast reply, you guys seem to be doing a great job maintaining this :smile:

**Comment by mzur** [comment]
That's right, workspace labels are not supported for GNOME 3.28.

**Comment by Laxen** [comment]
I've now upgraded to Gnome 3.34 with extension version 17 and everything works perfectly right away!

It could be nice to have the different features that are supported for each version listed on the [extension page](https://extensions.gnome.org/extension/1485/workspace-matrix/) so people don't get confused when something that's listed as a feature doesn't work on their version. I was running Ubuntu 18.04 LTS which is stuck on Gnome 3.28, so it's probably quite common that people run this version. I stepped up to Ubuntu 19.10 to get a newer Gnome and newer everything.

Thanks again for the super fast support and awesome extension :clap:
